src/Files.App/Views/Settings/TagsPage.xaml

```diff
@@ -24,19 +25,19 @@
 			<vc:StringToBrushConverter x:Key="StringToBrushConverter" />
 		</ResourceDictionary>
 	</Page.Resources>
+
 	<Page.DataContext>
-		<settingsviewmodels:TagsViewModel x:Name="ViewModel" />
+		<vm:TagsViewModel x:Name="ViewModel" />
 	</Page.DataContext>
 
-	<Grid>
+	<Grid Padding="16">
 		<StackPanel
 			HorizontalAlignment="Stretch"
 			VerticalAlignment="Stretch"
 			Spacing="4">
 			<StackPanel.ChildrenTransitions>
-				<TransitionCollection>
-					<EntranceThemeTransition />
-				</TransitionCollection>
+				<EntranceThemeTransition FromVerticalOffset="50" />
+				<RepositionThemeTransition IsStaggeringEnabled="False" />
```

Comment:
Let's split this into another PR, it's going to take me some time to test and see if I want to use the change.